GVRf/Framework/src/org/gearvrf/GVRCameraRig.java

```diff
@@ -145,7 +145,11 @@ public void setCameraSeparationDistance(float distance) {
      * @return The {@code float} value associated with {@code key}.
      */
     public float getFloat(String key) {
-        return NativeCameraRig.getFloat(getPtr(), key);
+        float[] value = NativeCameraRig.getFloat(getPtr(), key);
+        if (value == null) {
```

Comment:
Imho, I don't think it is necessary, but for code consistency, maybe you'd better use Preconditions.checkNotNull(). And also, I think it will help reduce the duplicate codes for every getX().

Thanks